Sources/FluentKit/Query/Builder/QueryBuilder.swift

```diff
@@ -83,27 +87,28 @@ public final class QueryBuilder<Model>
         return self.run()
     }
     
-    public func delete() -> EventLoopFuture<Void> {
+    public func delete(force: Bool = false) -> EventLoopFuture<Void> {
+        self.forceDelete = Model.init().deletedTimestamp == nil ? true : force
```

Comment:
I think this should be something like `shouldForceDelete`, then we do the actual check for `deletedTimestamp` in the run method. As is, having `forceDelete = false` on a model without timestamps is an invalid state that we could make impossible.